service_info/static/less/base/header/nav.less

```diff
@@ -83,6 +88,7 @@
           .align-items(center);
           .justify-content(space-between);
           .flex(0, 1, 100%);
+          flex-basis: 100% !important;
```

Comment:
This is the main addition of interest.

Elsewhere, the Less is set to reduce the percentage `flex-basis` values of flex elements because the IE flexbox implementation is buggy and makes them oversized, breaking layouts.

Here, though, it's essential that it be 100%—and seemingly IE is able to competently deal with `flex-basis:100%` elements.